.github/workflows/playwright.yml

```
@@ -0,0 +1,61 @@
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    
+    - name: Install dependencies in ${{ matrix.directory }}
+      uses: borales/actions-yarn@v4
+      with:
+        cmd: install
+      working-directory: ${{ matrix.directory }}
```

actionlint (1.7.7)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)